docs/websites/cms/install-and-configure-wordpress-using-wp-cli.md

```diff
@@ -0,0 +1,266 @@
+
+	cd /home/user
+	wget https://github.com/wp-cli/wp-cli/raw/master/utils/wp-completion.bash
+
+Now edit the .bash_profile so that it is loaded by the shell everytime you login.
+
+	nano ~/.bash_profile
+
+Add the following line in the editor.
+
+	source /<fullpath>/wp-completion.bash
+
+Assuming you downloaded the wp-completion.bash file in the home directory, the line to be added would become
+
+	source /home/user/wp-completion.bash
+
+Press **Ctrl+X** and then **Y** to save the changes.
+
+Now run the following command to reload the Bash profile.
+
+	source ~/.bash_profile
+
+That's it. Bash Completion is enabled.
+
+## Basics of WP-CLI
+Before we get into the helm of the things, let us learn some essential basics of how exactly WP-CLI works. This would help you feel comfortable with what is going to follow next.
+
+So far we have seen WP-CLI is accessed through the keyword `wp`. What follows that is a set of commands and their own sub commands. For example we have a command to download WordPress which goes like `wp core download`. Here wp is the main command while core and download are its sub-commands. There can be several sub-commands for each sub-command as well.
+
+WP-CLI also comes with a pretty detailed help section which will allow you to keep tab on all the commands you would need. You can access it by typing
+
+	wp help
+	
+Doing that and you will see a screen similar to
+
+	wp
+
+	DESCRIPTION
+
+	Manage WordPress through the command-line.
+
+	SYNOPSIS
+
+	wp <command>
+
+	SUBCOMMANDS
+
+	cache               Manage the object cache.
+	cap                 Manage user capabilities.
+	cli                 Get information about WP-CLI itself.
+	comment             Manage comments.
+	core                Download, install, update and otherwise manage WordPress proper.
+	cron                Manage WP-Cron events and schedules.
+	db                  Perform basic database operations.
+	eval                Execute arbitrary PHP code after loading WordPress.
+	eval-file           Load and execute a PHP file after loading WordPress.
+	:
+	
+	
+`:` is a prompt where you need to enter few commands to navigate through this help menu. Up and down arrow keys will take you through the complete help. And pressing q will exit the help menu. That's all you need to know for now. For complete detail on how to navigate through the complete help section, you can always type h at the above prompt.
+
+In our last step, we enabled Bash Completion feature for WP-CLI. To use that type `wp` and press tab twice. You would see the list of available commands. You can repeat the same by typing 'wp core' and then pressing tab twice. Now you will see a list of commands that can be used with `core`.
+
+## Installing WordPress
+
+### Seting up Database
+
+Before you proceed, you need to setup a database first. For that, login to the MySQL server first.
```

Comment:
Whenever there's a variable to change in a code block, announce it in the description above. In this case: Replace `user` with your MySQL user.